sdv/multi_table/base.py

```diff
@@ -253,6 +280,13 @@ def preprocess(self, data):
         processed_data = {}
         for table_name, table_data in data.items():
             synthesizer = self._table_synthesizers[table_name]
+            synthesizer.auto_assign_transformers(table_data)
+            foreign_key_columns = self._get_all_foreign_keys(table_name)
+            column_name_to_transformers = {
+                column_name: None
+                for column_name in foreign_key_columns
+            }
+            synthesizer.update_transformers(column_name_to_transformers)
```

Comment:
I think this is a fine solution but can we call attention to what we're doing here by pulling it out into a separate method called `_skip_foreign_key_transformations` or something like that?